tools/nnpackage_tool/model2nnpkg/model2nnpkg.sh

```diff
@@ -91,7 +91,7 @@ if [[ ${#configs_src[@]} -ne 0 ]] && [[ ${#configs_src[@]} -ne ${#models_src[@]}
 fi
 
 delim=""
-for modelpath in ${models_src[@]}
+for modelpath in "${models_src[@]}"
```

Comment:
https://stackoverflow.com/questions/8880603/loop-through-an-array-of-strings-in-bash#comment110320649_8880633